src/validation/constants.js

```diff
@@ -0,0 +1,44 @@
+export const TYPES = {
+  STRING: 'string',
+  NUMBER: 'number',
+  OBJECT: 'object',
+  BOOLEAN: 'boolean'
+};
+
+export const ERROR_MESSAGES = {
+  EMPTY_LABEL: 'Label cannot be empty',
+  EMPTY_SCHEMA: 'Schema must not be an empty object',
+  INVALID_TYPE: 'Value must be of type TYPE',
+  INVALID_SCHEMA: 'Invalid schema',
+  INVALID_NUMBER: 'Length cannot be negative',
+  INVALID_VALUE_TYPE: 'Invalid value type',
+  INVALID_SCHEMA_TYPE: 'Invalid schema type',
+  FORM_SCHEMA_MISMATCH: 'Schema and form do not match',
+  NO_MATCHING_PROPERTY: `No PROPERTY property to match`,
+  INVALID_MIN_OVER_MAX: 'Minimum length cannot be greater than the maximum',
+  EMPTY_PROPERTY: 'PROPERTY cannot be empty',
+  EMPTY_MATCHING_PROPERTY: 'Matching property cannot be empty',
+  INVALID_MIN_MAX: 'Minimum or maximum length cannot be less than the number of required characters'
+};
+
+export const VALIDATION_ERROR_MESSAGES = {
```

Comment:
are these also specific to oneleif, would be best to allow users to have their own error messages if possible

Reply:
This is the default for the package. I'm actually working on letting the user set their own error messages right now.